server/graphql/v1/mutations/orders.js

```diff
@@ -547,6 +547,7 @@ export async function createOrder(order, loaders, remoteUser, reqIp, userAgent,
         isEmbed: Boolean(order.context?.isEmbed),
         isGuest,
         isBalanceTransfer: order.isBalanceTransfer,
+        ...order.data,
```

Comment:
This should be at the top, just after `data: {` because we don't want people to override the other fields, otherwise an attacker could do:

```es6
createOrder({ data: { reqIp: "You'll never now!", isFeesOnTop: false, isGuest: false })
```

If possible, we should also add a `pick` to only allow the fields that we want to keep.